shared-bindings/ps2io/Ps2.c

```diff
@@ -68,21 +68,21 @@
 //|         ...
 //|
 STATIC mp_obj_t ps2io_ps2_make_new(const mp_obj_type_t *type, size_t n_args, size_t n_kw, const mp_obj_t *all_args) {
-    enum { ARG_datapin, ARG_clkpin };
+    enum { ARG_data_pin, ARG_clock_pin };
     static const mp_arg_t allowed_args[] = {
-        { MP_QSTR_datapin, MP_ARG_REQUIRED | MP_ARG_OBJ },
-        { MP_QSTR_clkpin, MP_ARG_REQUIRED | MP_ARG_OBJ },
+        { MP_QSTR_data_pin, MP_ARG_REQUIRED | MP_ARG_OBJ },
+        { MP_QSTR_clock_pin, MP_ARG_REQUIRED | MP_ARG_OBJ },
```

Comment:
anticipating this'll be v8 so incompatible changes are OK? I guess the docstring and the code didn't match before, but now they will...

Reply:
Yes, the docstring said `clock`, and the actual keyword was `clk`, so I chose `clock`. But no one complained about this, so I doubt if there's much code if any code that is using `clk`. So I consider this more of a bug fix. We have done similar fixes a few times for incorrect keyword args.